instrumentation/src/main.c

```diff
@@ -11,10 +11,11 @@
 static char *const allowed_env_vars[] = {ALLOWED_ENV_VARS};
 static size_t const allowed_env_vars_size = sizeof(allowed_env_vars) / sizeof(*allowed_env_vars);
 
+#define DOTNET_ENV_VAR_FILE "/etc/splunk/zeroconfig/dotnet.conf"
```

Comment:
Are there .NET-specific env vars that need be added to `ALLOWED_ENV_VARS` for this file?

Reply:
Yes, I've been chatting about it with Paulo who pointed me to this example:
https://github.com/signalfx/tracing-examples/blob/main/opentelemetry-tracing/opentelemetry-dotnet/automatic/aspnetcore-and-mongodb/InstrumentContainer/Dockerfile

We can use the env vars defined there to be allowed as a starting point.

Reply:
added those @pjanotti please check when you get a sec

Reply:
You can remove OTEL_DOTNET_AUTO_INTEGRATIONS_FILE, this one is deprecated - I will update the example.